packages/web-runtime/src/container/bootstrap.ts

```diff
@@ -52,17 +52,16 @@ export const announceConfiguration = async (path: string): Promise<RuntimeConfig
  *
  * @param vue
  * @param runtimeConfiguration
- * @param store
+ * @param language
  */
 export const announceStore = async ({
   vue,
-  runtimeConfiguration,
-  store
+  runtimeConfiguration
 }: {
   vue: VueConstructor
   runtimeConfiguration: RuntimeConfiguration
-  store: Store<any>
-}): Promise<void> => {
+}): Promise<any> => {
+  const store = new Store({ ...storeOptions })
```

Comment:
if storeOptions are deep i would prefer to use Object.assign (shallow clone)

Reply:
`...` is shallow as well AFAIK. We just moved this part basically, it was like this before as well.